### PR TITLE
`General`: Add documentation for team exercises

### PR DESCRIPTION
**Pull request by manuelmanso**
#### General
- [ ] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

Add page about `Team exercises` to documentation.

### Description

Added a new page for `Team exercises`, and added links to it in the Readme and in the About us page.

Reviews would be appreciated on, particularly on if I should go more in depth in some aspects, if I missed anything important, and where the page should go (I wasn't too sure where to put it but decided to put it alongside the differente exercise types).

### Steps for Testing

* Ensure that the link to the documentation page works correctly on the About Us page and in the Readme
* Checkout the new documentation page (link above)

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### CODE REVIEW

**Review by ge65cer — DISMISSED** [review]
Found some missing images.

Screenshot 2022-07-14 at 08 29 18

**Review by Marvin-Vitcu — DISMISSED** [review]
Tested on TS1. Works as intended.

For me, this first sentence is quite confusing. I would rephrase it to:
When creating an exercise, the instructor first needs to select the mode "Team".

image

**Review by luisa-we — APPROVED** [review]
The documentation looks good to me.
Also tested on TS1, the link works as expected.

**Review by alaraozdenler — DISMISSED** [review]
Tested on ts2, the link on About us didn't work for me
image

**Comment by ge65cer** [comment]
@alaraozdenler The link can not work, as it points to the production documentation and this PR is not merged yet.

**Comment by Marvin-Vitcu** [comment]
> Tested on ts2, the link on About us didn't work for me image

This is the correct behaviour. The link work as soon as this PR is merged

**Comment by manuelmanso** [comment]
> Tested on TS1. Works as intended.
> 
> For me, this first sentence is quite confusing. I would rephrase it to: When creating an exercise, the instructor first needs to select the mode "Team".
> 
> image

Thanks, I rephrased it, I also think it sounds better

**Review by babyygemperor — DISMISSED** [review]
Tested on ts1, works as described

**Review by Marvin-Vitcu — DISMISSED** [review]

**Review by nairaarnez — APPROVED** [review]
Very good documentation! While reading it I came up with some rephrasing ideas: 

image
I would change this to: To manually create a team, instructors

image
to: Instructors can use the "Export teams" button to export the teams from an exercise to a file. This can then be imported into other exercises to facilitates the process of creating teams. To import the teams in a new exercise, instructors can use the "Import teams" button. 

image
The same team can work on multiple exercises. Viewing...

**Comment by nairaarnez** [comment]
I realized I read too much while reading Lectures, I'll leave this other rephrasing ideas in the comments since I already had them:

image
to: by clicking on the tab labeled Lectures, students can see all published lectures of a course grouped by date.

image
...to the following page. Here students can see the title, descriptio, units and attachments of the lecture.

image
...site, by clicking on "Lectures", the following page opens, where it is posible to manage the lectures

**Review by luisa-we — APPROVED** [review]
Tested on TS2, still works as expected.
